src/components/TextField/TextField.tsx

```diff
@@ -24,12 +25,16 @@ export const TextField: React.FC<Props> = ({
   required = false,
   onChange = () => {},
 }) => {
-  // generage a unique id once on component load
   const [id] = useState(() => `${name}-${getRandomDigits()}`);
 
-  // To show errors only if the field was touched (onBlur)
   const [touched, setTouched] = useState(false);
   const hasError = touched && required && !value;
```

Comment:
```suggestion
  const hasError = touched && required && !value.trim();
```
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/86923089/2adc434d-fb54-4a26-8392-39e841172b40)